### ISSUE TITLE
ENA-231: Make setting switches colorblind friendly

### ISSUE THREAD

**Issue by DNin01** [LLK/scratch-gui#8577]
### The problem
On setting switches that use buttons with icons, such as the shown/hidden switch or the rotation style setting, each icon's active state is colored in, and each icon's inactive state is grayed out. But looking at it in grayscale, you actually can't tell which one is on. See the following image:
![Image of the shown/hidden switch in color compared to an image of it in grayscale](https://user-images.githubusercontent.com/106490990/196005751-2c5dfdd1-8538-42ce-a5d6-222c73703a76.png)
For colorblind users, this can be an issue.

### The solution
We shouldn't rely on colored-in versus black-and-white to show which option was selected. Instead, we could change the background color when an option is selected, and make it white, not gray, when the option isn't selected, like so:
![The Show button has a blue background, and the Hide button has a white background.](https://user-images.githubusercontent.com/106490990/196005804-8eb0d6ea-98fc-4fd0-a4c7-b46a6f42d1f7.png)
This difference is easier for colorblind users to see.

**Comment by Secret-chest** [comment]
Wow! Even for normal users, this would be easier to see too!

**Comment by DNin01** [comment]
This problem also affects the rotation style icons in the popup for changing a sprite's direction.

**Comment by CST1229** [comment]
Also, note that the paint editor already highlights the selected tool like this:
![](https://user-images.githubusercontent.com/68464103/209718746-d82b4342-c832-4bfc-9faf-61e42e1dd716.png)

**Comment by chocolate-pie** [comment]
### I tried to make that.
![スクリーンショット 2023-02-20 152749](https://user-images.githubusercontent.com/106949016/220031477-b285c3bb-ef47-45fa-abbf-2ce97c72558b.png)
**patch (diff.patch)** 
```patch
diff --git a/src/components/sprite-info/sprite-info.css b/src/components/sprite-info/sprite-info.css
index 93ca30f..7b7f919 100644
--- a/src/components/sprite-info/sprite-info.css
+++ b/src/components/sprite-info/sprite-info.css
@@ -52,6 +52,9 @@
     height: 100%;
 }
 
+.icon.is-active {
+    filter: brightness(0) invert(1);
+}
 /* @todo: refactor radio divs to input */
 .radio-wrapper {
     white-space: nowrap; /* make sure visibilty buttons don't wrap */
@@ -63,6 +66,7 @@
 }
 .radio.is-active {
     filter: none;
+    background-color: $motion-primary;
 }
 .radio.is-disabled {
     cursor: default;
diff --git a/src/components/sprite-info/sprite-info.jsx b/src/components/sprite-info/sprite-info.jsx
index 09c94d0..b6fbfb6 100644
--- a/src/components/sprite-info/sprite-info.jsx
+++ b/src/components/sprite-info/sprite-info.jsx
@@ -201,7 +201,10 @@ class SpriteInfo extends React.Component {
                                 onKeyPress={this.props.onPressVisible}
                             >
                                 <img
-                                    className={styles.icon}
+                                    className={classNames(styles.icon, {
+                                        [styles.isActive]: this.props.visible && !this.props.disabled,
+                                        [styles.isDisabled]: this.props.disabled
+                                    })}
                                     src={showIcon}
                                 />
                             </div>
@@ -220,7 +223,10 @@ class SpriteInfo extends React.Component {
                                 onKeyPress={this.props.onPressNotVisible}
                             >
                                 <img
-                                    className={styles.icon}
+                                    className={classNames(styles.icon, {
+                                        [styles.isActive]: !this.props.visible && !this.props.disabled,
+                                        [styles.isDisabled]: this.props.disabled
+                                    })}
                                     src={hideIcon}
                                 />
                             </div>
```
